configure.ac

```diff
@@ -282,6 +282,17 @@ if test "x$bitfury" = xyes; then
 fi
 AM_CONDITIONAL([HAS_BITFURY], [test x$bitfury = xyes])
 
+bitfury16="no"
+
+AC_ARG_ENABLE([bitfury16],
+	[AC_HELP_STRING([--enable-bitfury16],[Compile support for BitFury 16nm ASICs (default disabled)])],
+	[bitfury16=$enableval]
+	)
+if test "x$bitfury16" = xyes; then
+	AC_DEFINE([USE_BITFURY16], [1], [Defined to 1 if BitFury 16nm ASIC support is wanted])
+	drivercount=x$drivercount
```

Comment:
yep, you are right
I've marked my device as standalone, as it is run on specific hardware